test/config/test_config.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import os
+import json
+from datetime import datetime
+from src.config import config
+from src.config import BoundsResult
+
+
+@pytest.fixture
+def empty_config():
+    return config
+
+@pytest.fixture
+def loaded_config(tmp_path):
```

Comment:
I see that you used this fixture for checking bounds, can you also write a test that assert get_value for this fixture?